tests/test_deprecation.py

```diff
@@ -40,7 +40,7 @@ def test_deprecation_warnings_are_accompanied_by_appropriate_comment(
             contents = f.read()
 
         matches: List[str] = regex_for_deprecation_comments.findall(contents)
-        warning_count: int = contents.count("DeprecationWarning")
+        warning_count: int = contents.count("DeprecationWarning(")
```

Comment:
Why this change to add the opening param?

Reply:
This ended up being a mistake.